templates/index.html

```diff
@@ -0,0 +1,243 @@
+        <a href="https://opencollective.com/kivy"
+          class="text-sm md:text-base font-light md:pr-2 xl:pr-5 pt-3 pb-5 underline">
+          <b>See all the {{organizations.length}} organizations supporting Kivy on OpenCollective</b>
+        </a>
+      </div>
+
+      <p class="w-full text-xl font-light text-center pt-10 pb-5">
+        {{individuals.length}} <b>individuals</b> are <b>supporting Kivy</b>
+      </p>
+      <a v-bind:href="'https://www.opencollective.com/' + individual.account.slug"
+        v-bind:title="individual.account.name + ', Total contributed: ' + individual.totalDonations.value + ' ' + individual.totalDonations.currency"
+        v-for="individual in individuals">
+        <div class="bg-[#e8e8e8] w-8 h-8 rounded-full flex justify-center items-center">
+          <img class="h-full w-full object-cover rounded-full" v-bind:src="individual.account.imageUrl" />
+        </div>
+      </a>
+      <div class="w-full flex text-center justify-end">
+        <a href="https://opencollective.com/kivy"
+          class="text-sm md:text-base font-light md:pr-2 xl:pr-5 pt-3 pb-5 underline">
+          <b>See all the {{individuals.length}} individuals supporting Kivy on OpenCollective</b>
+        </a>
+      </div>
+    </div>
+  </div>
+
+  [[ include "components/support_kivy_banner.html" ]]
+  [[ include "components/need_help.html" ]]
+  [[ include "components/footer.html" ]]
+
+  <script src="https://unpkg.com/vue@3"></script>
+  <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+  <script>
+    let opencollective_graphql_url = "https://api.opencollective.com/graphql/v2";
+    const graphql_query =
+        `
+        query fetchContributors($account_type: [AccountType]!, $profile_image_height: Int!){
+          account(slug: "kivy") {
+            name
+            slug
+            members(role: BACKER, accountType: $account_type, limit:9999999) {
+              nodes {
+                account {
+                  id
+                  slug
+                  name
+                  imageUrl(height:$profile_image_height, format: jpg)
+                }
+                tier {
+                  name
+                }
+                totalDonations {
+                  value
+                  currency
+                }
+              }
+            }
+          }
+        }`;
+
+    function filter_duplicated_profiles(arr) {
+      return arr.filter((value, index, self) =>
+        index === self.findIndex((t) => (
+          t.account.id === value.account.id
+        ))
+      )
+    }
+
+    const { createApp } = Vue;
+    var opencollective_app = createApp({
+      el: '#opencollective_app',
+      data() {
+        return {
+          organizations: [],
+          individuals: []
+        };
+      },
+      computed: {
+      },
+      mounted() {
+        let self = this;
+
+        axios({
+          url: 'https://api.opencollective.com/graphql/v2',
```

Comment:
We probably wanted to leverage `opencollective_graphql_url` variable instead

Reply:
Fixed during the migration to `content/static/js/opencollective_app.js`